.golangci.yaml

```diff
@@ -43,6 +44,15 @@ linters-settings:
   exhaustive:
     default-signifies-exhaustive: true
 
+  goheader:
+    template: |-
+      Copyright {{ DATES }} Crunchy Data Solutions, Inc.
+
+      SPDX-License-Identifier: Apache-2.0
+    values:
+      regexp:
+        DATES: '((201[7-9]|202[0-3]) - 2024|2024)'
```

Comment:
Our policy is also to use the date range. I was worried about trying to do numbers in regex, but this actually looks okay to me. Nothing before `2017`, and when the year kicks over, we slide the `2024` left:

````diff
-        DATES: '((201[7-9]|202[0-3]) - 2024|2024)'
+        DATES: '((201[7-9]|202[0-4]) - 2025|2025)'
````

Reply:
> it would bug us if we changed the copyright in, say, late December...

Yeah. This is less about reminding us to change the year and more about spelling it right.